packages/nimble-components/src/dialog/tests/dialog.stories.ts

```diff
@@ -19,6 +19,7 @@ interface DialogArgs {
     content: ExampleContentType;
     show: undefined;
     close: undefined;
+    largeMode: boolean;
```

Comment:
We should add a case to  `dialog-matrix.stories.ts` that configures this attribute. The matrix story is used by Chromatic for visual comparison tests, so this will provide coverage on the PR build to ensure this configuration continues to behave as expected.